source/frontend/src/features/contacts/formModels.ts

```diff
@@ -183,7 +183,7 @@ export class IEditableOrganizationForm {
   alias?: string;
   incorporationNumber?: string;
   comment?: string;
-  isDisabled: boolean;
+  isDisabled: string | boolean = false;
```

Comment:
This field should **always** be string. Yes, it will contain string values of "true" or "false" but those are string representations of boolean values. We need this field to be a string because it is used in Formik to render the "disabled" dropdown (SELECT input) which always takes string values (so no boolean values are accepted for select)

The pattern we follow is to use the utility method **booleanToString()** to convert from the API value (which is boolean) into the string representation that the dropdown can render. We do this in the **fromApi()** method.

Then in the corresponding **toApi()** method we call the "inverse" function - **stringToBoolean()** to convert the string representation from formik into the API boolean value.

From what I see the code in contact is doing this half-way. The "toApi" is correctly converting from string to boolean. BUT the "fromApi" is missing the conversion from a boolean. The solution is to make "isDisabled" here a string and follow the pattern from "UpdatePropertyDetailsFormModel.ts" below:

![image](https://github.com/user-attachments/assets/60a753b2-b2bb-433e-9030-f8751a13ab1c)

Reply:
Thank you for the comments and lead me to a better solution. I'm updating this PR with the suggested changes. Thank you!!!!